pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echoit "  > Adding possible values 0, 1 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-storage-engine" ]; then
+    echoit "  > Adding possible value InnoDB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-row-metadata" ]; then
+    echoit "  > Adding possible value full and minimal for option '${OPTION}' to the final list..."
+    echo "${OPTION}=full" >> ${OUTPUT_FILE}
+    echo "${OPTION}=minimal" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-transaction-dependency-tracking" ]; then
+    echoit "  > Adding possible value COMMIT_ORDER , WRITESET and WRITESET_SESSION for option '${OPTION}' to the final list..."
+    echo "${OPTION}=COMMIT_ORDER" >> ${OUTPUT_FILE}
+    echo "${OPTION}=WRITESET" >> ${OUTPUT_FILE}
+    echo "${OPTION}=WRITESET_SESSION" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--caching-sha2-password-public-key-path" ]; then
+    echoit "  > Adding possible value public_key.pem for option '${OPTION}' to the final list..."
+    echo "${OPTION}=public_key.pem" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-tmp-storage-engine" ]; then
+    echoit "  > Adding possible value InnoDB  for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--ft-boolean-syntax" ]; then
+    echoit "  > Adding possible values + - > < ( ) ~ * : " " & | for option '${OPTION}' to the final list..."
+    echo "${OPTION}=+ -><()~*:""&|" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--group-replication-consistency" ]; then
+    echoit "  > Adding possible value EVENTUAL , BEFORE for option '${OPTION}' to the final list..."
+    echo "${OPTION}=BEFORE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=EVENTUAL" >> ${OUTPUT_FILE}
+    echo "${OPTION}=BEFORE_ON_PRIMARY_FAILOVER" >> ${OUTPUT_FILE}
+    echo "${OPTION}=AFTER" >> ${OUTPUT_FILE}
+    echo "${OPTION}=BEFORE_AND_AFTER" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-buffer-pool-filename" ]; then
+    echoit "  > Adding possible value ib_buffer_pool for option '${OPTION}' to the final list..."
+    echo "${OPTION}=ib_buffer_pool" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-compress-debug" ]; then
+    echoit "  > Adding possible values none , zlib , lz4 , lz4hc for option '${OPTION}' to the final list..."
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+    echo "${OPTION}=zlib" >> ${OUTPUT_FILE}
+    echo "${OPTION}=lz4" >> ${OUTPUT_FILE}
+    echo "${OPTION}=lz4hc" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-segment-reserve-factor" ]; then
+    echoit "  > Adding possible values 0.03 , 12.5 , 20 , 30 and 40 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0.03" >> ${OUTPUT_FILE}
+    echo "${OPTION}=12.5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=20" >> ${OUTPUT_FILE}
+    echo "${OPTION}=30" >> ${OUTPUT_FILE}
```

Comment:
better to test for the value 39. boundary value testing

Reply:
Tested and worked with value 39.

Reply:
I asked you to rename 30 to 39.